pkg/test/framework/components/istio/flags.go

```diff
@@ -24,6 +24,9 @@ func init() {
 		"Specifies the namespace where the istiod resides in a typical deployment. Defaults to istio-system")
 	flag.StringVar(&settingsFromCommandline.TelemetryNamespace, "istio.test.kube.telemetryNamespace", settingsFromCommandline.TelemetryNamespace,
 		"Specifies the namespace in which kiali, tracing providers, graphana, prometheus are deployed.")
+	flag.StringVar(&settingsFromCommandline.TrustedZtunnelNamespace, "istio.test.kube.trustedZtunnelNamespace",
+		settingsFromCommandline.TrustedZtunnelNamespace,
+		"Specifies the trusted namespace for ztunnel. Used in Ambient mode.")
```

Comment:
Since ztunnel is used only in Ambient mode, we can remove the following sentence.
```suggestion
		"Specifies the namespace in which ztunnel is deployed.")
```